store/search/ts.go

```diff
@@ -38,72 +40,88 @@ type storeImplWithMetrics struct {
 	s Store
 }
 
+func (m *storeImplWithMetrics) measure(ctx context.Context, name string, f func(ctx context.Context) error) {
+	// Low level measurement wrapper that is called by the measure functions on the appropriate receiver
+	tags := metrics.GetSearchTags(ctx, name)
+	if config.DefaultConfig.Metrics.Search.ResponseTime {
+		metrics.SearchRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets)
+		defer metrics.SearchRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets).Start().Stop()
+	}
+	if config.DefaultConfig.DatadogTrace.Enabled {
+		parentSpan, exists := tracer.SpanFromContext(ctx)
+		if exists {
+			spanOpts := []tracer.StartSpanOption{
+				tracer.ServiceName("tigris.search"),
```

Comment:
Consider using const for these and encapsulating the defintions in a package.

Reply:
Done.